READMEs/c2d-flow.md

```diff
@@ -23,7 +23,18 @@ Let's go through each step.
 ## 1. Setup
 
 ### First steps
-
+Use the `--with-c2d` option when running barge in order to include the Compute-to-Data backend
+```console
+./start_ocean.sh --with-c2d
+```
+When running barge with c2d enabled, use the following bash script to wait until
```

Comment:
This should be split in 2: there should be a wait script only for ocean-contracts in datatokens flow. And here there should be a wait script for both ocean-contracts and ocean-c2d. It is explained right here, and the explanation should also be updated.